week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -11,9 +11,16 @@ The output of running your code should be:
 
 class ShoppingCart {
   // Add your code here
+  constructor() {
+    this.allItems = [];
+  }
 
+  addItem(items) {
```

Comment:
You've used a variable `items` here but you could use `item` singular if you're only adding one item at a time